app/src/main/java/mx/yellowme/youst/pokemon/PokemonListLoader.kt

```diff
@@ -18,17 +21,15 @@ object PokemonListLoader {
         view.setState(ChallengeWithListActivity.ListState.LOADING)
         call.enqueue(object : Callback<MutableList<Pokemon>> {
             override fun onResponse(
-                call: Call<MutableList<Pokemon>>, response: Response<MutableList<Pokemon>>
+                    call: Call<MutableList<Pokemon>>, response: Response<MutableList<Pokemon>>
```

Comment:
Was this space added by the reformat code tool?

Reply:
I will remove the initial space. I miss to format this file 🙌